JS/lib/cards.js

```diff
@@ -0,0 +1,142 @@
+//   <section id="question-card0">
+//   <div class="question-card__total">
+//     <h2>QUESTION X</h2>
+
+//     <p>Can you die without perishing ?</p>
+
+//     <button class="question-card__answer">Show Answer</button>
+
+//     <i class="fas fa-bookmark question-card__bookmark-non"></i>
+//     <p class="answer">
+//       When your body leaves your soul, your soul will be forever
+//     </p>
+//   </div>
+// </section>
+
+export function createCard(card) {
+  const cardSection = document.createElement('section')
+
+  const headlineFrame = document.createElement('div')
+  cardSection.append(headlineFrame)
+  headlineFrame.classList.add('question-card__total-headline-frame')
+
+  const headLine = document.createElement('h2')
+  headLine.textContent = 'QUESTION'
+  headlineFrame.append(headLine)
+  headLine.classList.add('question-card__total-headline')
+
+  const cardLayout = document.createElement('div')
+  cardSection.append(cardLayout)
+  cardLayout.classList.add('question-card__total')
```

Comment:
I find it a little bit confusing reading, that you append an element and then in the next line you're adding something to the element. 
In my head is it like: The elements is created, it is appended...okay done, to the next thing...and then there is more.

It is nothing wrong about it, but like I'm said, it is confusing me to read it like that. ;-)

```suggestion
  cardLayout.classList.add('question-card__total')
  cardSection.append(cardLayout)
```

Reply:
Interesting. this was my line of thought: create, append, add a class (like that div).